<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text Edit</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>271</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="print_btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Print Text Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Text Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="font_btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Font</string>
   </property>
  </widget>
  <widget class="QPushButton" name="italic_btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Font Italic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="red_btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Font Color Red</string>
   </property>
  </widget>
  <widget class="QPushButton" name="size_up_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Size Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="size_down_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Size Down</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
